pkg/ddl/backfilling_operators.go

```diff
@@ -666,11 +664,11 @@ func NewWriteExternalStoreOperator(
 			for i := range indexes {
 				builder := external.NewWriterBuilder().
 					SetOnCloseFunc(onClose).
-					SetKeyDuplicationEncoding(hasUnique).
 					SetMemorySizeLimit(memoryQuota).
 					SetTiKVCodec(tikvCodec).
 					SetBlockSize(blockSize).
-					SetGroupOffset(i)
+					SetGroupOffset(i).
+					SetOnDup(onDuplicateKey)
```

Comment:
```suggestion
					SetOnDupAction(onDuplicateKey)
```

Reply:
@tangenta all other places are using `onDup`, i think it's ok without the `Action` suffix

Reply:
@CbcWestwolf

Reply:
ok